skrf/network.py

```diff
@@ -3223,33 +3223,31 @@ def step_response(self, window='hamming', n=None, pad=1000):
         y : class:`numpy.ndarray`
             Step response
 
+        Raises
+        ------
+        ValueError
+            If used with an Network with more than one port
+
+        NotImplementedError
+            If used with non equidistant sampled frequency vector
+
         See Also
         -----------
             impulse_response
             extrapolate_to_dc
         """
-        if self.nports != 1:
-            raise ValueError('Only one-ports are supported')
```

Comment:
why did you remove that part?

Reply:
This constraint is checked in the impulse_response function.

Reply:
ha yes, OK